tools/prepare_msd.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Details about MSD datasets: http://medicaldecathlon.com/
+
+The file structure is as following:
+lung_coronavirus
+|--20_ncov_scan.zip
+|--infection.zip
+|--lung_infection.zip
+|--lung_mask.zip
+|--lung_coronavirus_raw
+│   ├── 20_ncov_scan
+│   │   ├── coronacases_org_001.nii.gz
+│   │   ├── ...
+│   ├── infection_mask
+│   ├── lung_infection
+│   ├── lung_mask
+├── lung_coronavirus_phase0
+│   ├── images
+│   ├── labels
+│   │   ├── coronacases_001.npy
+│   │   ├── ...
+│   │   └── radiopaedia_7_85703_0.npy
+│   ├── train_list.txt
+│   └── val_list.txt
```

Comment:
please edit the comment with the msd dir here.

Reply:
It's changed. MSD all have similar structure. Used Task 01 as an example